dali/operators/decoder/nvjpeg/nvjpeg_decoder_decoupled_api.h

```diff
@@ -48,7 +48,7 @@
 
 namespace dali {
 
-using ImageInfo = EncodedImageInfo<int>;
```

Comment:
This is a fallout from "replace all" - we can revert it...

Reply:
Removed then.